logger/src/lib.rs

```diff
@@ -282,34 +230,23 @@ pub struct Logger {
     show_level: AtomicBool,
     show_file_path: AtomicBool,
     show_line_numbers: AtomicBool,
-    level_info: LevelInfo,
-    // Used in case we want to send logs to a FIFO.
-    log_fifo: Mutex<Option<PipeLogWriter>>,
-    // Used in case we want to send metrics to a FIFO.
-    metrics_fifo: Mutex<Option<PipeLogWriter>>,
+    level: AtomicUsize,
+    // Human readable logs will be outputted here.
+    log_buf: Mutex<Option<Box<dyn Write + Send>>>,
```

Comment:
Nit: I would create a type synonym `Destination` for `Box<dyn Write + Send>`, or something like that, so that the meaning of this type is clearer.

Reply:
I think that would be a confusing obfuscation. `Box<dyn Write + Send>` is much clearer in purpose.

Reply:
I also think it is more obvious this way and thus clearer.